src/lib/jsonParseV2/jsonRequestTreat.cpp

```diff
@@ -78,7 +78,7 @@ std::string jsonRequestTreat
     if ((answer = parseDataP->ent.res.check(ciP, EntitiesRequest)) != "OK")
     {
       OrionError error(SccBadRequest, answer);
-      return error.render(ciP, "");
+      return setStatusCodeAndSmartRender(ciP, error);
```

Comment:
Fixed in 3e4cd945bb1ab64ca9dc5507568445f5241b8c32. I think the result is better.